cdap-proto/src/main/java/io/cdap/cdap/proto/operation/OperationMeta.java

```diff
@@ -80,4 +81,66 @@ public Instant getCreateTime() {
   public Instant getEndTime() {
     return endTime;
   }
+
+  /**
+   * Creates a Builder.
+   */
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  /**
+   * Create a Builder from existing run.
+   *
+   * @param meta existing meta to copy fields from
+   */
+  public static Builder builder(OperationMeta meta) {
+    return new Builder(meta);
```

Comment:
can create the new builder and call the .setXYZ() methods here, to make sure the logic will always be consistent

Reply:
done